tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Tests\E2E\Services\Sites;
+
+use Tests\E2E\Scopes\ProjectCustom;
+use Tests\E2E\Scopes\Scope;
+use Tests\E2E\Scopes\SideServer;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Validator\Datetime as DatetimeValidator;
+
+class SitesCustomServerTest extends Scope
+{
+    use SitesBase;
+    use ProjectCustom;
+    use SideServer;
+
+    public function testCreateSite(): array
+    {
+        /**
+         * Test for SUCCESS
+         */
+        $site = $this->createSite([
+            'siteId' => ID::unique(),
+            'name' => 'Test',
+            'framework' => 'sveltekit',
+            'installCommand' => 'npm install --force',
+            'buildCommand' => 'npm run build',
+            'outputDirectory' => './build',
+            'buildRuntime' => 'node-22',
+            'serveRuntime' => 'static-1',
+            'subdomain' => 'test'
+        ]);
+
+        $siteId = $site['body']['$id'] ?? '';
+
+        $dateValidator = new DatetimeValidator();
+        $this->assertEquals(201, $site['headers']['status-code']);
+        $this->assertNotEmpty($site['body']['$id']);
+        $this->assertEquals('Test', $site['body']['name']);
+        $this->assertEquals('sveltekit', $site['body']['framework']);
+        $this->assertEquals(true, $dateValidator->isValid($site['body']['$createdAt']));
+        $this->assertEquals(true, $dateValidator->isValid($site['body']['$updatedAt']));
+        $this->assertEquals('npm install --force', $site['body']['installCommand']);
+        $this->assertEquals('npm run build', $site['body']['buildCommand']);
+        $this->assertEquals('./build', $site['body']['outputDirectory']);
+        $this->assertEquals('node-22', $site['body']['buildRuntime']);
+        $this->assertEquals('static-1', $site['body']['serveRuntime']);
+
+        $variable = $this->createVariable($siteId, [
+            'key' => 'siteKey1',
+            'value' => 'siteValue1',
+        ]);
+        $variable2 = $this->createVariable($siteId, [
+            'key' => 'siteKey2',
+            'value' => 'siteValue2',
+        ]);
+        $variable3 = $this->createVariable($siteId, [
+            'key' => 'siteKey3',
+            'value' => 'siteValue3',
+        ]);
+
+        $this->assertEquals(201, $variable['headers']['status-code']);
+        $this->assertEquals(201, $variable2['headers']['status-code']);
+        $this->assertEquals(201, $variable3['headers']['status-code']);
+
+        return [
+            'siteId' => $siteId,
+        ];
+    }
+
+    /**
+     * @depends testCreateSite
```

Comment:
Lets rewrite tests in this file to have no dependencies. 
If you need to list, or get, or delete, inside the test you can make new site.

this way we prevent need to use @depends

Reply:
In that case, we'd also need to delete the resources we create in each test at the end of it so that it doesn't cause conflicts with other tests.
For example, while testing `listSites`, if I am creating sites for it inside the test itself, it should not list sites from other tests. Do you agree each test should also perform cleanup then?